CMakeLists.txt

```diff
@@ -38,6 +38,18 @@ set(KWIVER_BINARY_DIR           "${CMAKE_BINARY_DIR}")
 set(KWIVER_CONFIG_FILE          "${KWIVER_BINARY_DIR}/kwiver-config.cmake")
 # Create initial setup script
 set(KWIVER_SETUP_SCRIPT_FILE    "${KWIVER_BINARY_DIR}/setup_KWIVER.sh" )
+if(WIN32)
```

Comment:
Maybe the best compromise is the always construct the .bat but only have a the `if(WIN32)` around the install step.  That way we can check that the files is constructed properly on a Linux machine, but it doesn't clutter up the install with a file that is useless on that platform.